src/mono/cmake/options.cmake

```diff
@@ -56,6 +56,7 @@ option (ENABLE_OVERRIDABLE_ALLOCATORS "Enable overridable allocator support")
 option (ENABLE_SIGALTSTACK "Enable support for using sigaltstack for SIGSEGV and stack overflow handling, this doesn't work on some platforms")
 option (USE_MALLOC_FOR_MEMPOOLS "Use malloc for each single mempool allocation, so tools like Valgrind can run better")
 option (STATIC_COMPONENTS "Compile mono runtime components as static (not dynamic) libraries")
+option (DISABLE_WASM_USER_THREADS "Disable creation of managed threads, only allow runtime internal threads")
```

Comment:
update the message to mention that this only applies to wasm

Reply:
Updated.  also clarified that runtime internal managed threads will be allowed (I need them to support the EventPipe counter sampling thread).